src/tools/tidy/src/target_policy.rs

```diff
@@ -21,6 +21,7 @@ const EXCEPTIONS: &[&str] = &[
     "xtensa_esp32s2_espidf",
     "xtensa_esp32s3_none_elf",
     "xtensa_esp32s3_espidf",
+    "i586_pc_nto_qnx700", // Renamed to i686-pc-nto-qnx700, see https://github.com/rust-lang/rust/issues/136495
```

Comment:
Why is this necessary?

Reply:
This was necessary as otherwise the test and the end failed (I got the `missing assembly test` error message).

Reply:
Maybe tidy uses the bootstrap rustc to determine the list of targets it expects tests for

Reply:
I thought so too but https://github.com/rust-lang/rust/pull/137367 seems to work so